delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/StoragesConfig.scala

```diff
@@ -211,8 +211,10 @@ object StoragesConfig {
       showLocation: Boolean,
       defaultMaxFileSize: Long,
       defaultBucket: String,
-      prefix: Uri
-  ) extends StorageTypeEntryConfig
+      prefix: Option[Uri]
+  ) extends StorageTypeEntryConfig {
+    val prefixUri: Uri = prefix.getOrElse(Uri.Empty)
```

Comment:
I'm not sure what the motivation for this is, but it feels weird that we have two fields available in the config, but we would only ever use one of them. Perhaps it's possible to specify the default to be an empty string, and have the parser deal with that correctly?